src/_includes/components/pagination.njk

```diff
@@ -2,26 +2,31 @@
 {% set numberOfPages = pagination.pages.length %}
 {% set pageNumberIndex = pagination.pageNumber + 1 %}
 
-<nav class="pagination" aria-label="Pagination links">
-  {% if pagination.href.previous %}
-    <a href="{{ pagination.href.previous }}" data-direction="backwards">
+<nav aria-label="Pagination links" class="pagination">
+  <a class="button pagination__button pagination__button--prev" data-direction="backwards" {% if pageNumberIndex === 1 %}disabled{% endif %} href="{{ pagination.href.previous }}">
```

Comment:
Not sure if `disabled` will have the intended effect here, but we could `tabindex="-1"` to omit from the interaction order, and maybe even `aria-disabled/hidden="true"` 🤔 if we want to hinge styling off that? ([ref](https://getbootstrap.com/docs/4.0/components/pagination/#disabled-and-active-states), [ref](https://a11y-style-guide.com/style-guide/section-navigation.html#kssref-navigation-pagination))

I see you've got `.button[disabled]` to handle the styling. Might need to piggy back that for something that's anchor appropriate, because the `pointer-events` etc are nice.

Reply:
Agreed, implemented all of the above.